docs/sdk-and-tools/overview.md

```diff
@@ -163,6 +163,16 @@ Events notifier is an external service that can be used to fetch block events an
 
 [comment]: # (mx-context-auto)
 
+### Chain simulator
+
+Chain simulator is designed to replicate the behavior of a local testnet.
```

Comment:
... It can also be pre-initialized / initialized with blockchain state from other networks, such as mainnet. Or something similar.

Reply:
added